src/aat/resources/features/F-044/F-044.feature

```diff
@@ -0,0 +1,136 @@
+@F-044
+Feature: F-044: Submit event creation as Case worker
+
+  Background: Load test data for the scenario
+    Given an appropriate test context as detailed in the test data source
+
+  @S-278
+  Scenario: must submit the event creation successfully for correct inputs
+    Given a user with [an active profile in CCD]
+    And a case that has just been created as in [Standard_Full_Case_Creation_Data]
+    And a successful call [to get an event token for the case just created] as in [F-044-Prerequisite]
+    When a request is prepared with appropriate values
+    And the request [contains a case Id for just created case]
```

Comment:
It is bit confusing, may be like this: 
contains a case that has just been created as in Standard_Full_Case_Creation_Data